src/app/pages/give/give.js

```diff
@@ -202,6 +214,12 @@ export default class Give extends CMSPageController {
         if (invalid) {
             event.preventDefault();
             this.update();
+        } else {
+            localStorage.setItem('donation:first_name', this.el.querySelector('[name=First_Name]').value);
```

Comment:
I used the names in the form where the values are entered for the local storage names. I added the donation: prefix so they did not clash with any other items in local storage. The salesforce form names are determined by Salesforce. Which would you like me to match?